tests/test_fingerprint.py

```diff
@@ -0,0 +1,7 @@
+from datasets.fingerprint import Hasher
+
+
+def test_dill():
+    # AttributeError: module 'dill._dill' has no attribute 'stack'
+    hasher = Hasher()
+    hasher.update(lambda x: x)
```

Comment:
I think you can just check this
```suggestion
    Hasher.hash(lambda x: x)
```